### ISSUE TITLE
Create a release?

### ISSUE THREAD

**Issue by thejpster** [probe-rs/hs-probe#17]
Would it be possible to create some releases in Github, containing the rendered schematics and gerbers? I think this would more easily allow people to track the revisions of the PCB, without the git repository spiralling in size due to lots of PDFs being uploaded.

You can even use Github Actions to create the PDFs and the Gerbers. See https://github.com/Neotron-Compute/Neotron-Baseboard-100/blob/develop/.github/workflows/kicad-release.yml for a basic example.

**Comment by Yatekii** [comment]
Yes, this is something we have on the todo :) I just didn't get around to do this yet. Thanks for making an issue :)
